framework/src/components/logger/logger.js

```diff
@@ -58,7 +65,7 @@ const createLogger = ({ fileLogLevel, consoleLogLevel, logFileName }) => {
 			: [];
 	const streams = [...consoleStream, ...fileStream];
 	return bunyan.createLogger({
-		name: 'lisk-framework',
+		name: loggerName,
```

Comment:
everything except this new `loggerName` has default value, so probably it should provide default, too.
Also, createLogger with `loggerName` seems redundant, although it kinda applies to other property too 😢